src/Lykke.Service.LiteCoin.API.Services/Wallet/WalletBalanceService.cs

```diff
@@ -58,10 +58,11 @@ public class WalletBalanceService:IWalletBalanceService
             if (wallet != null)
             {
                 var balance = await _blockChainProvider.GetBalanceSatoshiFromUnspentOutputs(wallet.Address, _confirmationsSettings.MinConfirmationsToDetectOperation);
+                var lastBlock = await _blockChainProvider.GetLastBlockHeight();
```

Comment:
Is this call atomic with previous GetBalanceSatoshiFromUnspentOutputs()? Not looks so. Can we do these to actions atomic?